pkg/controller/controller_utils.go

```diff
@@ -130,8 +130,12 @@ func GetPushgatewayImage(cluster *v1alpha1.TidbCluster) string {
 }
 
 // PDMemberName returns pd member name
-func PDMemberName(clusterName string) string {
-	return fmt.Sprintf("%s-pd", clusterName)
+func PDMemberName(clusterName string, setName ...string) string {
+	if len(setName) == 0 || setName[0] == "" {
```

Comment:
What would cause the slice `setName ` to be non-empty but the first element is empty?

Reply:
The [main ](https://github.com/pingcap/tidb-operator/pull/265/files#diff-d70b6a464792e09ec86d28a6584a9249R89) PD's [name](https://github.com/pingcap/tidb-operator/pull/265/files#diff-d70b6a464792e09ec86d28a6584a9249R110) is `""`